.github/workflows/helm-release.yaml

```diff
@@ -0,0 +1,30 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'charts/**'
+      - '!**.md'
+
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.2.1
```

Comment:
Could you tell me how this action work?
We are publishing mdbook documents already to `gh-pages` branch.
Will this conflict or not?

Reply:
Fixed to avoid conflicts.
https://github.com/cybozu-go/accurate/pull/11/commits/a4ca3e580f1927788bfc2a13507962a249a89225

I tested it in my repository:
https://github.com/d-kuro/accurate/commits/gh-pages